### PR TITLE
change authFile to local path

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

File: docs/en_US/TrainingService/PaiMode.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMod
 * shmMB
     * Optional key. Set the shmMB configuration of OpenPAI, it set the shared memory for one task in the task role.
 * authFile
-    * Optional key, Set the auth file path for private registry while using PAI mode, [Refer](https://github.com/microsoft/pai/blob/2ea69b45faa018662bc164ed7733f6fdbb4c42b3/docs/faq.md#q-how-to-use-private-docker-registry-job-image-when-submitting-an-openpai-job).
+    * Optional key, Set the auth file path for private registry while using PAI mode, [Refer](https://github.com/microsoft/pai/blob/2ea69b45faa018662bc164ed7733f6fdbb4c42b3/docs/faq.md#q-how-to-use-private-docker-registry-job-image-when-submitting-an-openpai-job), the value should be a valid local authFile path.
```
[the remark this answers is not in the document]
Comment (SparkSnail) [reply docs/en_US/TrainingService/PaiMode.md]: fixed.

File: src/nni_manager/training_service/pai/paiTrainingService.ts [hunk 2]
```diff
@@ -436,7 +436,20 @@ class PAITrainingService implements TrainingService {
                 this.paiTrialConfig.shmMB,
             )
         ];
+        // Upload authFile to hdfs
+        let authFileHdfsDir: string | undefined = undefined;
```
Comment (QuanluZhang) [inline src/nni_manager/training_service/pai/paiTrainingService.ts]: I think this logic should be included in setclustermetadata, but this field is within trialconfig....

File: docs/en_US/TrainingService/PaiMode.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMod
 * shmMB
     * Optional key. Set the shmMB configuration of OpenPAI, it set the shared memory for one task in the task role.
 * authFile
-    * Optional key, Set the auth file path for private registry while using PAI mode, [Refer](https://github.com/microsoft/pai/blob/2ea69b45faa018662bc164ed7733f6fdbb4c42b3/docs/faq.md#q-how-to-use-private-docker-registry-job-image-when-submitting-an-openpai-job).
+    * Optional key, Set the auth file path for private registry while using PAI mode, [Refer](https://github.com/microsoft/pai/blob/2ea69b45faa018662bc164ed7733f6fdbb4c42b3/docs/faq.md#q-how-to-use-private-docker-registry-job-image-when-submitting-an-openpai-job), you can prepare the authFile and simply provide the local path of this file, NNI will upload this file to HDFS for you.
```
Comment (liuzhe-lz) [inline docs/en_US/TrainingService/PaiMode.md]: Link to master branch?

File: src/nni_manager/training_service/pai/paiTrainingService.ts (hunk 2 above)
Comment (SparkSnail) [reply src/nni_manager/training_service/pai/paiTrainingService.ts]: fixed.

**Pull request merged**